spikingjelly/activation_based/neuron/adapt.py

```
@@ -271,23 +271,25 @@ def multi_step_forward(self, x_seq: torch.Tensor):
+                    self.v_threshold,
+                    self.v_reset,
+                    self.v_rest,
+                    self.a,
+                    self.b,
+                    self.tau_w,
+                    self.v_c,
+                    self.a0,
+                    self.detach_reset,
+                    self.surrogate_function,
+                )
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)